upstreams.go

```diff
@@ -129,8 +137,8 @@ func (u *Upstreams) keepUpdated(ctx caddy.Context, cli *client.Client) {
 			select {
 			case <-messages:
 				debounced(func() {
-					containers, err := cli.ContainerList(ctx, container.ListOptions{
-						Filters: filters.NewArgs(filters.Arg("label", LabelEnable)),
+					containers, err := cli.ContainerList(ctx, types.ContainerListOptions{
```

Comment:
Did you use the old version of docker? I believe `types.ContainerListOptions` is deprecated.